infrahub_sdk/ctl/cli_commands.py

```diff
@@ -392,11 +398,176 @@ def protocols(
 
 @app.command(name="version")
 @catch_exception(console=console)
-def version(_: str = CONFIG_PARAM) -> None:
-    """Display the version of Infrahub and the version of the Python SDK in use."""
+def version() -> None:
+    """Display the version of Python and the version of the Python SDK in use."""
 
-    client = initialize_client_sync()
-    response = client.execute_graphql(query="query { InfrahubInfo { version }}")
+    console.print(f"Python: {platform.python_version()}\nPython SDK: v{sdk_version}")
 
-    infrahub_version = response["InfrahubInfo"]["version"]
-    console.print(f"Infrahub: v{infrahub_version}\nPython SDK: v{sdk_version}")
+
+@app.command(name="info")
+@catch_exception(console=console)
+def info(detail: bool = typer.Option(False, help="Display detailed information."), _: str = CONFIG_PARAM) -> None:  # noqa: PLR0915
+    """Display the status of the Python SDK."""
+
+    query = """
+query GET_PROFILE_DETAILS {
+  AccountProfile {
+    id
+    display_label
+    account_type {
+      value
+      __typename
+      updated_at
+    }
+    status {
+      label
+      value
+      updated_at
+      __typename
+    }
+    description {
+      value
+      updated_at
+      __typename
+    }
+    label {
+      value
+      updated_at
+      __typename
+    }
+    member_of_groups {
+      count
+      edges {
+        node {
+          display_label
+          group_type {
+            value
+          }
+          ... on CoreAccountGroup {
+            id
+            roles {
+              count
+              edges {
+                node {
+                  permissions {
+                    count
+                    edges {
+                      node {
+                        display_label
+                        identifier {
+                          value
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+            display_label
+          }
+        }
+      }
+    }
+    __typename
+    name {
+      value
+      updated_at
+      __typename
+    }
+  }
+}
+"""
+
+    info: dict[str, Any] = {
+        "error": None,
+        "status": ":x:",
+        "infrahub_version": "N/A",
+        "user_info": {},
+        "groups": {},
+    }
+    try:
+        client = initialize_client_sync()
```

Comment:
Would be good to move this code into specific functions that other users could reuse outside of the CLI
```python
client.get_version()
client.get_user_permissions()
```

or similar